### ISSUE TITLE
Link to WCAG 2.1 Understanding document

### ISSUE THREAD

**Issue by yatil** [karlgroves/overlayfactsheet#29]
Currently the conformance section links to the WCAG 2.0 version of Understanding.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance

We probably should link to the latest 2.1 version, which – theoretically – should be updated more regularly:

https://www.w3.org/WAI/WCAG21/Understanding/conformance

(And yes, the 2.1 page has the same language, referring to 2.0, which is a known issue. So maybe we don’t actually want to make this change? https://github.com/w3c/wcag/issues/1068)

**Comment by karlgroves** [comment]
I agree!
